bciers/apps/administration/app/bceidbusiness/industry_user/select-operator/(request-access)/page.tsx

```diff
@@ -0,0 +1,23 @@
+"use client";
+import SelectOperatorForm from "../../../../components/userOperators/SelectOperatorForm";
+import { useSession } from "next-auth/react";
+import getUserFullName from "@bciers/utils/getUserFullName";
+export default function Page() {
+  const { data: session } = useSession();
+  const names = getUserFullName(session)?.split(" ");
+
+  return (
+    <>
+      <section className="text-center my-auto text-2xl flex flex-col gap-3 mx-auto">
+        <p>
+          Hi,{" "}
+          <b>
+            {names?.[0]} {names?.[1]}!
+          </b>
+        </p>
+        <p>Which operator would you like to log in to?</p>
+      </section>
+      <SelectOperatorForm />
+    </>
+  );
```

Comment:
Style nit - 
This should likely be a div/section and group everything together as it is causing some misalignment and also buggy zoom behaviour:

Should be grouped together more and centered at 90% zoom (used 90 instead of 100 for demonstration purposes because the issue is exaggerated a bit more):
<img width="1469" alt="Screenshot 2024-09-04 at 12 01 55 PM" src="https://github.com/user-attachments/assets/977c44e4-faf3-4279-bdfb-ef731dd8e37a">


The next step looks nice at 90% zoom:
<img width="1469" alt="Screenshot 2024-09-04 at 12 02 09 PM" src="https://github.com/user-attachments/assets/689a3b44-1761-4402-b47f-af0157b8a23c">


When I zoom out:
<img width="1300" alt="Screenshot 2024-09-04 at 11 46 45 AM" src="https://github.com/user-attachments/assets/6d0545fe-e9de-45f6-a082-a949d4d57681">

The next step zoomed out:
<img width="1300" alt="Screenshot 2024-09-04 at 11 55 56 AM" src="https://github.com/user-attachments/assets/11b91b24-f58c-40b4-8576-e53026c0b13a">

Reply:
@marcellmueller 

🪄 Magic

 Thanks! :)